modules/sdk-coin-apt/src/lib/transaction/transaction.ts

```diff
@@ -258,14 +267,24 @@ export abstract class Transaction extends BaseTransaction {
   }
 
   loadInputsAndOutputs(): void {
+    const totalAmount = this._recipients?.reduce(
```

Comment:
`_recipients` will be initialized with empty list in the constructor.
```suggestion
    const totalAmount = this._recipients.reduce(
```